### ISSUE TITLE
Update C21 Trademark in Global Footer Header

### ISSUE THREAD

**Issue by juanmriosv** [Century21Stores/GLOBAL#93]
Red, stylized "C21" trademark to be updated in the upper/header region of the Global Footer, in the line of copy that reads "Be the First to Receive C21 Exclusives".

Development for: [C21 Trademark Sketch (UXD-10)](https://github.com/Century21Stores/UXD/issues/10)
[WorkFront Request](https://c21.my.workfront.com/issue/view?ID=596d1967007c0119bef0343f8c68eeb9)
**End Result:**
Waiting for final Comp with Size and spacing

**Comment by juanmriosv** [comment]
To built
![screen shot 2017-08-29 at 11 36 53 am](https://user-images.githubusercontent.com/16789090/29831914-9de15cc2-8cb4-11e7-9245-663204cf23a3.png)

**Comment by juanmriosv** [comment]
On Staging

**Comment by juanmriosv** [comment]
Footer on c21stores.com sent for review.
Brickwork in progress (after above approved)

**Comment by AdenaG** [comment]
Seems like the requirements were just to replace "C21" in the footer, but waiting on Workfront ticket access to verify for QA.

**Comment by juanmriosv** [comment]
Yes, that was the requirement. I believe Sonia will be able to provide access to the Workfront.

**Comment by AdenaG** [comment]
Yep, already requested from Sonia, been doing that for Workfront tickets to whomever is the requestor, so nothing needed on your end :).

In that case, pass in Staging.
